repository-data/development/src/main/resources/hcm-content/content/documents/lks/news.yaml

```diff
@@ -0,0 +1,406 @@
+
+          <p>Health Education England is responsible for providing training places for GP specialty training recruits and managing the specialist training process.</p>
+
+          <p>Through the Choose GP campaign Health Education England is recruiting foundation year doctors as well as locally employed/ trust grade doctors and consultants who want to retrain as a GP.</p>
+
+          <p>Figures from NHS Digital show there are 8,664 GPs in training grade in GP placements across England as of September 2021.</p>
+
+          <p><strong>Professor Simon Gregory DL, Deputy Medical Director, Primary and Integrated Care</strong>&nbsp;at&nbsp;<strong>Health Education England, said:</strong>&nbsp;“It has been a challenging 18 months for the profession across the board. As a GP myself it is heart-warming to see so many trainees wanting to become family doctors.</p>
+
+          <p>“The fact that so many people are interested in joining general practice speciality training means we are doing all we can to help provide a sufficient and sustainable workforce.”</p>
+
+          <p><strong>Professor Wendy Reid, Director of Education and Quality and National Medical Director at Health Education England, said:</strong> “We are pleased that we have managed to hit the new target of recruiting 4,000 new GPs.</p>
+
+          <p>“We are committed at Health Education England to promoting a career as a GP for trainee doctors approaching the end of their foundation training.</p>
+
+          <p>“Recent GP workforce figures show that trainees are a valuable part of the primary care workforce. On a daily basis they see and treat patients and provide a great service across the country.”</p>
+
+          <p>ENDS</p>
+      /hee:heroImage:
+        jcr:primaryType: hippogallerypicker:imagelink
+        hippo:docbase: cafebabe-cafe-babe-cafe-babecafebabe
+        hippo:facets: []
+        hippo:modes: []
+        hippo:values: []
+      /hee:rightHandBlocks[1]:
+        jcr:primaryType: hee:contactCardReference
+        /hee:content:
+          jcr:primaryType: hippo:mirror
+          hippo:docbase: a352e94a-1e98-4280-abc5-b0853ae32e26
+      /hee:rightHandBlocks[2]:
+        jcr:primaryType: hee:ctaCardReference
+        /hee:ctaCardContentBlock:
+          jcr:primaryType: hippo:mirror
+          hippo:docbase: c9d58420-ae91-42a8-a19d-82c7f78a3bd8
+  /call for-nhs-ambassadors to-inspire-young-people:
+    jcr:primaryType: hippo:handle
+    jcr:mixinTypes: ['hippo:named', 'hippo:versionInfo', 'mix:referenceable']
+    jcr:uuid: 1f0fe8c9-06cd-4cfd-9ab1-1e616f43e372
+    hippo:name: Call for NHS Ambassadors to inspire young people
+    hippo:versionHistory: 964e4b72-ce18-405f-810c-d2b2db78c63a
+    /call for-nhs-ambassadors to-inspire-young-people[1]:
+      jcr:primaryType: hee:news
+      jcr:mixinTypes: ['mix:referenceable']
+      jcr:uuid: 4f988973-ca43-4f09-b5b2-99ede28d74f2
+      hee:author: Sam Sparks
+      hee:categories: [add-key1, add-key2]
```

Comment:
Could we get rid of these old categories as it is creating empty comma in the listing:

![Screenshot_2021-12-23_at_08_31_17](https://user-images.githubusercontent.com/35143542/147212067-63e8928d-14f0-4eeb-9382-18e6d0bb9958.jpg)

Reply:
Done